### PR TITLE
feat: implement multi-transfer

### PR DESCRIPTION
**Pull request by Teja2045**
## Summary by CodeRabbit

- **New Features**
  - Implemented a new `Messages` component for displaying a list of messages with pagination.
  - Added a `MultiTxUpload` component for handling file uploads and parsing transactions.
  - Introduced a `CustomMultiLineTextField` component for multiline text input fields.

- **Enhancements**
  - Updated the `SendPage` to use new custom text fields from `customTextFields.json`.
  - Enhanced the `MultiTransfer` component with form handling and state management.
  - Improved message navigation and deletion options in the `Message` component.

- **Bug Fixes**
  - Fixed an issue where the `multiTxns` function was not handling transaction dispatch correctly.

- **Documentation**
  - Updated documentation to reflect new components and hooks usage.

- **Refactor**
  - Refactored `useGetChainInfo` hook to include `getOriginDenomInfo` function for retrieving denomination information.

- **Style**
  - Added new CSS classes for improved styling and layout of components.

- **Tests**
  - No specific test-related changes mentioned.

- **Chores**
  - Removed unused imports and functions from `send.ts` to streamline the codebase.

- **Revert**
  - No reverts mentioned.

### CODE REVIEW

File: frontend/src/app/(routes)/transfers/components/MultiTxUpload.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+              const file = e.target.files[0];
+              if (!file) {
+                return;
+              }
+
+              const reader = new FileReader();
+              reader.onload = (e) => {
+                if (!e.target) return null;
+                const contents = e.target.result;
+                onFileContents(contents);
```
[the remark this answers is not in the document]
Comment (Teja2045) [reply frontend/src/app/(routes)/transfers/components/MultiTxUpload.tsx:94]: Its handled already somewhere else

File: frontend/src/app/(routes)/transfers/components/Messages.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+              MULTI_TRANSFER_MSG_COUNT * index,
+              MULTI_TRANSFER_MSG_COUNT * index + MULTI_TRANSFER_MSG_COUNT
+            )
+            .map((msg, index, array) => (
+              <div key={index}>
+                <Message msg={msg} onDelete={onDelete} index={index} />
+                {index !== array.length - 1 && (
+                  <div style={{ marginBottom: '28px' }} />
+                )}
+              </div>
```
[the remark this answers is not in the document]
Comment (Teja2045) [reply frontend/src/app/(routes)/transfers/components/Messages.tsx]: it doesnt have unique value

File: frontend/src/txns/bank/send.ts [hunk 3]
```diff
@@ -27,5 +26,5 @@ export function serialize(msg: Msg): string {
   const { toAddress, amount } = msg.value;
   return `sent ${amount[0].amount} ${
```
Comment (aleem1314) [inline frontend/src/txns/bank/send.ts:27]:
```suggestion
  return `Send ${amount[0].amount} ${
```

**Review by aleem1314 — CHANGES_REQUESTED** [review]
transfer-1

**Pull request merged**